docker/package/tezos_baking_wizard.py

```diff
@@ -0,0 +1,489 @@
+    print("Progress: ", percent, "%, ", int(done / (1024 * 1024)), "MB", end="\r")
+
+
+def pprint_table(options):
+    result = "\n"
+    i = 1
+    str_format = "{:1}. {:<20}  {}\n"
+    for o in options:
+        description = textwrap.indent(textwrap.fill(options[o], 60), " " * 25).lstrip()
+        result += str_format.format(i, o, description)
+        i += 1
+
+    return result
+
+
+def color(input, colorcode):
+    return colorcode + input + "\x1b[0m"
+
+
+class Step:
+    def __init__(
+        self,
+        id: str,
+        prompt: str,
+        help: str,
+        default: str = "1",
+        options=None,
+        validator=None,
+        reqs=lambda _vs: True,
+    ):
+        self.id = id
+        self.prompt = prompt
+        self.help = help
+        self.default = default
+        self.options = options
+        self.validator = validator
+        self.reqs = reqs
+
+
+def run_wizard(networks):
+    print("Tezos Baking Wizard")
+    print()
+    print(
+        "Welcome to a wizard helping you to set up the infrastructure for interacting with the Tezos blockchain."
+    )
+    print(
+        "In order to run a baking instance, you'll need the following Tezos binaries:",
+        "tezos-client, tezos-node, tezos-baker-<proto>, tezos-endorser-<proto>.",
+    )
+    print(
+        "To access help and possible options for each question, type in 'help' or '?'."
+    )
+    print("Type in 'exit' to quit.")
+    print()
+
+    modes = {
+        "baker": "Set up and start all services for baking: "
+        "tezos-node, tezos-baker, and tezos-endorser.",
+        "node": "Only bootstrap and run the Tezos node.",
+    }
+
+    snapshot_import_modes = {
+        "download rolling": "Import rolling snapshot from xtz-shots.io (recommended)",
+        "download full": "Import full snapshot from xtz-shots.io",
+        "file": "Import snapshot from a file",
+        "url": "Import snapshot from a url",
+        "skip": "Skip snapshot import and synchronize with the network from scratch",
+    }
+
+    key_import_modes = {
+        "ledger": "From a ledger",
+        "secret-key": "Either the unencrypted or password-encrypted secret key for your address",
+        "json": "Faucet JSON file from https://faucet.tzalpha.net/",
+    }
+
+    values = {}
+
+    steps = [
+        Step(
+            id="network",
+            prompt="Which Tezos network would you like to use?\nCurrently supported:\n",
+            help="The selected network will be used to set up all required services.\n"
+            "The currently supported protocol is 009-PsFLoren (used on florencenet and mainnet).\n"
+            "Also, note that the corresponding packages have protocol suffix in lowercase, "
+            "e.g. the list of available baker packages can be found on Serokell's launchpad, i.e. here:\n"
+            "https://launchpad.net/~serokell/+archive/ubuntu/tezos/+packages?field.name_filter=tezos-baker&field.status_filter=published"
```

Comment:
I'm not sure that this is useful information here, if one is using this script has at least install some package and the script does not instruct (or run commands) to install more, so... :man_shrugging: